### PR TITLE
Loading Data

### PR DESCRIPTION
**Pull request by sarahmish**
Added an intermediate function to load .csv files, making the loader independent of the format. This can later be expanded to include another medium.

### CODE REVIEW

**Comment by Hector-hedb12** [comment]
I just added a minor comment regarding docs. This looks good to me.

File: cardea/featurization/featurization.py [hunk 1]
```diff
@@ -3,20 +3,9 @@
 
 class Featurization():
     """"A class that generates a feature matrix from its attributes.
```
[the remark this answers is not in the document]
Comment (sarahmish) [reply cardea/featurization/featurization.py]: Fixed in the latest commit "c9a5311"

File: tests/cardea/featurization/test_featurization.py [hunk 2]
```diff
@@ -55,22 +55,10 @@ def cutoff():
...
-
-
-def test_featurization_target(featurization, cutoff):
-    assert featurization.target == 'Encounter'
-
-
-def test_generate_feature_matrix(featurization):
-    fm_encoded, features_encoded = featurization.generate_feature_matrix()
+def test_generate_feature_matrix(featurization, entityset, cutoff):
+    fm_encoded, features_encoded = featurization.generate_feature_matrix(entityset, "Encounter", cutoff)
```
[the remark this answers is not in the document]
Comment (sarahmish) [reply tests/cardea/featurization/test_featurization.py]: Flake is not complaining, so I left it as is.

[the remark this answers is not in the document]
Comment (Hector-hedb12) [reply tests/cardea/featurization/test_featurization.py]: yes, I agree. This comment was made 5 days ago and therefore it is `outdated`

File: cardea/featurization/featurization.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hector-hedb12) [reply cardea/featurization/featurization.py]: great!

**Comment by Hector-hedb12** [comment]
This looks good to me @sarahmish but there is a small merge conflict.

**Comment by Hector-hedb12** [comment]
Thanks @sarahmish . Ready to merge @bryanfarris

**Pull request merged**